transport-file/src/main/java/no/difi/meldingsutveksling/transport/FileTransport.java

```diff
@@ -26,29 +24,34 @@
 public class FileTransport implements Transport {
 
     @Override
-    public void send(ApplicationContext context, EduDocument eduDocument) {
+    public void send(ApplicationContext context, StandardBusinessDocument sbd) {
         String fileName = createFilename();
-        ModelMapper mapper = new ModelMapper();
-
-        no.difi.meldingsutveksling.noarkexchange.schema.receive.StandardBusinessDocument toWrite =
-                mapper.map(eduDocument, no.difi.meldingsutveksling.noarkexchange.schema.receive.StandardBusinessDocument.class);
 
         File f = new File(fileName);
         try {
-            JAXBContext jaxbContext = JAXBContextFactory.createContext(new Class[]{EduDocument.class, Payload.class, Kvittering.class}, null);
+            JAXBContext jaxbContext = JAXBContextFactory.createContext(new Class[]{StandardBusinessDocument.class, Payload.class, Kvittering.class}, null);
             Marshaller jaxbMarshaller = jaxbContext.createMarshaller();
             jaxbMarshaller.setProperty(Marshaller.JAXB_FORMATTED_OUTPUT, true);
-            jaxbMarshaller.marshal(new ObjectFactory().createStandardBusinessDocument(eduDocument), new FileOutputStream(f));
-            JAXBElement<no.difi.meldingsutveksling.noarkexchange.schema.receive.StandardBusinessDocument> element =
-                    new JAXBElement<>(new QName("ns"),
-                            no.difi.meldingsutveksling.noarkexchange.schema.receive.StandardBusinessDocument.class, toWrite);
+            jaxbMarshaller.marshal(new ObjectFactory().createStandardBusinessDocument(sbd), new FileOutputStream(f));
         } catch (JAXBException e) {
             throw new MeldingsUtvekslingRuntimeException("file write error ", e);
         } catch (FileNotFoundException e) {
             throw new MeldingsUtvekslingRuntimeException();
         }
     }
 
+    /**
+     * Not currently in use.
+     *
+     * @param context
+     * @param sbd     An sbd with a payload consisting of metadata only
+     * @param is      InputStream pointing to the encrypted ASiC package
+     */
+    @Override
+    public void send(ApplicationContext context, StandardBusinessDocument sbd, InputStream is) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1186)